src/synthesizer/parametric/morphology.py

```diff
@@ -385,3 +385,64 @@ def compute_density_grid_from_arrays(self, xx, yy, units=kpc):
                 "Only kpc and milliarcsecond (mas) units are supported "
                 "for morphologies."
             )
+
+
+class Gaussian2D(MorphologyBase):
```

Comment:
This looks good but you need to have a `compute_density_from_array` method to match the others above. I'll make some changes to ensure this errors for you if you don't.